source/examples/rapids-optuna-hpo/notebook.ipynb

```diff
@@ -855,6 +318,22 @@
     "print(\"Number of finished trials: \", len(study.trials))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You should see logs like the following.\n",
+    "\n",
+    "```text\n",
+    "[I 2024-08-06 09:41:40,161] Trial 1 finished with value: 8.238207899472073 and parameters: {'C': 40.573838784392514, 'penalty': 'l2', 'fit_intercept': True}. Best is trial 1 with value: 8.238207899472073.\n",
+    "... \n",
+    "[I 2024-08-06 09:41:58,423] Trial 143 finished with value: 8.210414278942531 and parameters: {'C': 0.3152731188939818, 'penalty': 'l1', 'fit_intercept': True}. Best is trial 52 with value: 8.205579602300705.\n",
+    "\n",
+    "Best params: {'C': 1.486491072441749, 'penalty': 'l2', 'fit_intercept': True}\n",
+    "Number of finished trials:  144\n",
+    "```"
+   ]
```

Comment:
The logs generated above:

* are huge
* contain stacktraces with absolute paths from wherever the code was run

Proposing here to instead clear the output of that cell and just keep this shorter snippet of the logs in a markdown block.